examples/00-fluent/modeling_ablation.py

```diff
@@ -0,0 +1,441 @@
+""".. _modeling_ablation:
+
+Modeling Ablation
+-------------------------------------------
+"""
+
+#######################################################################################
+# Objective
+# =====================================================================================
+#
+# Ablation is an effective treatment used to protect an atmospheric reentry vehicle from
+# the damaging effects of external high temperatures caused by shock wave and viscous
+# heating. The ablative material is chipped away due to surface reactions that remove a
+# significant amount of heat and keep the vehicle surface temperature below the melting
+# point. In this tutorial, Fluent ablation model is demonstrated for a reendtry vehicle
+# geometry simplified as a 3D wedge.
+#
+# This tutorial demonstrates how to do the following:
+#
+# * Define boundary conditions for a high-speed flow.
+# * Set up the ablation model to model effects of a moving boundary due to ablation.
+# * Initiate and solve the transient simulation using the density-based solver.
+#
+# Problem Description:
+# ====================
+#
+# The geometry of the 3D wedge considered in this tutorial is shown in following figure.
+# The air flow passes around a nose of a re-entry vehicle operating under high speed
+# conditions. The inlet air has a temperature of 4500 K, a gauge pressure of 13500 Pa,
+# and a Mach number of 3. The domain is bounded above and below by symmetry planes
+# (displayed in yellow). As the ablative coating chips away, the surface of the wall
+# moves. The moving of the surface is modeled using dynamic meshes. The surface moving
+# rate is estimated by Vieille's empirical law:
+#
+# where r is the surface moving rate, p is the absolute pressure, and A and n are model
+# parameters. In the considered case, A = 5 and n = 0.1.
+
+
+####################################################################################
+# .. math::
+#
+#    r = A \cdot p^n
+
+
+# %%
+# .. image:: ../../_static/ablation-problem-schematic.png
+#    :align: center
+#    :alt: Problem Schematic
```

Comment:
@prmukherj  Images associated with each example script are missing. e.g. `ablation-problem-schematic.png`. All of them should be uploaded into `_static` directory.

Reply:
Done, Thank you.